STEP_Project/popup.js

```diff
@@ -0,0 +1,120 @@
+var pageType = undefined;
+
+/** Enum for page types. */
+const PAGE_TYPES = {
+  ORG_LIST_PAGE: 0,
+  SETTINGS_PAGE: 1, // includes onoff page
+  OTHER: 2,
+};
+
+(function()
+{
+  checkTabUrl();
+  addRemoveButton = document.getElementById("addRemove");
+  injectContent(addRemoveButton);
+}()
+);
+
+/**
+ * Checks url of the current tab to detect settings or OU list page
+ * @param {string} givenurl - url of current page
+ */
+function findPagecase(givenurl)
+{
+  let orgunitspage = givenurl.match(/admin.google.com\/u\/[0-9]+\/ac\/orgunits/g);
+  let settingspage = givenurl.match(/admin.google.com\/u\/[0-9]+\/ac\/appsettings\/[0-9]+\/.+/g);
+  let onoffpage = givenurl.match(/admin.google.com\/u\/[0-9]+\/ac\/settings\/serviceonoff.*/g);
```

Comment:
It is possible for the first user to not have "u/0" in the URL. In that case https://admin.google.com/ac/appsettings/725740718362/videoSettings and https://admin.google.com/u/0/ac/appsettings/725740718362/videoSettings both work and I believe the current regex does not capture that.

Reply:
This change was made in the next PR of adding a content script.